twitchio/ext/eventsub/server.py

```diff
@@ -52,7 +52,7 @@ async def delete_all_active_subscriptions(self):
         for subscription_id in active_subscriptions:
             await self.delete_subscription(subscription_id)
 
-    async def get_subscriptions(self, status: str = None):
+    async def get_subscriptions(self, status: str = None, sub_type: str = None, user_id: str = None):
```

Comment:
both of these are Optional, user_id should be Optional[int] at a user facing level to remain consistent with the rest of the lib. status should technically also be Optional[str] to actually pass type checking.

Reply:
don't forget to cast back to `str` on the user_id in the underlying method